infra/src/dsl/output-generators/map-to-helm-values.ts

```diff
@@ -75,7 +75,7 @@ const serializeService: SerializeMethod<HelmService> = async (
       maxUnavailable: 1,
     },
     healthCheck: {
-      port: serviceDef.healthPort,
+      // port: serviceDef.healthPort,
```

Comment:
We shouldn't commit commented code

Reply:
oops, thanks

and it should not have been commented out 😄